node/src/indexer.rs

```diff
@@ -91,7 +91,7 @@ async fn handle_block(
                             tracing::warn!("`sign` did not produce entropy");
                             continue;
                         }
-                        let Ok(entropy) = serde_json::from_str::<'_, [u8; 32]>(&receipt.logs()[0])
+                        let Ok(entropy) = serde_json::from_str::<'_, [u8; 32]>(&receipt.logs()[1])
```

Comment:
hmm, why are we taking the second log entry instead of the first? What is the difference and can you show an example?

Reply:
I've added `"sign: signer={}, payload={:?} path={:?}"` log. Now entropy is the second log.